mypy.ini

```diff
@@ -13,3 +13,7 @@ explicit_package_bases = True
 disable_error_code =
 	# Disable due to many false positives
 	overload-overlap,
+
+# Different error on Windows Python 3.8 than on other platforms
+[mypy-test.*]
+ignore_missing_imports = True
```

Comment:
Sorry I'm just now getting around to this one. Given the mention of Python 3.8, I think it may be obsolete. I'll get main stable, and then test without these lines.

Reply:
No hurry, there's a ton of these PRs, feel free to ping me once updated !